client/src/components/Header/Header.css

```diff
@@ -0,0 +1,30 @@
+h1 {
+  font-size: large;
+}
+
+.channel-info {
```

Comment:
Responsiveness and other features are already built into the Bulma CSS framework that we are using. We don't need to manually add. Otherwise, there is no point in using a CSS framework as a base.

Reply:
Done using bulma css